compiled-templates/templates/base.html

```diff
@@ -0,0 +1,146 @@
+                        
+                        
+                            <li><a href="#lists-example">lists</a></li>
+                        
+                        
+                            <li><a href="#tables-example">Tables</a></li>
+                        
+                        
+                            <li><a href="#forms-example">Form elements</a></li>
+                        
+                        
+                            <li><a href="#notifications-example">Notification styles</a></li>
+                        
+                    </ul>
+                    
+
+                
+            </section>
+
+            <section class="components">
+                <h2 class="section-title">Components</h2>
+
+                
+                    <div class="colors-section" id="colors-example">
+                        <h3>Colors</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="buttons-section" id="buttons-example">
+                        <h3>Buttons</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="icons-section" id="icons-example">
+                        <h3>Icons</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="typography-section" id="typography-example">
+                        <h3>Typography</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="lists-section" id="lists-example">
+                        <h3>Lists</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="tables-section" id="tables-example">
+                        <h3>Tables</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="forms-section" id="forms-example">
+                        <h3>Forms</h3>
+                        
+                    </div>
+                
+
+                
+                    <div class="notifications-section" id="notifications-example">
+                        <h3>Notifications</h3>
+                        
+                    </div>
+                
+            </section>
+
+            
+                <section class="ux-elements">
+                    <h2 class="section-title">UX Elements</h2>
+                    
+                </section>
+            
+
+        </main>
+
+        <footer class="page-footer">
+            
+                <div class="structure">
+                    <p class="copyright">Incuna-style-guide maintained by <a href="https://github.com/incuna">Incuna Ltd.</a></p>
+                </div>
+            
+        </footer>
+
+    </body>
+
+
+</html>
```

Comment:
I wasn't sure, I guess it's probably better not to, I can add it to my list of things to do. I guess this is the first drawback of swig it doesn't remove the blank space when it compiles